.github/workflows/langwatch-app-ci.yml

```
@@ -56,15 +72,73 @@ jobs:
+
+      - name: Prepare files
+        working-directory: langwatch
+        run: npm run start:prepare:files
+
+      - name: Run integration tests
         working-directory: langwatch
-        run: ./node_modules/.bin/tsgo --noEmit --project ./tsconfig.json
+        run: npm run test:integration -- --run --bail=1 --poolOptions.forks.singleFork
 
-  build:
+   build:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 3

(indentation)